src/read.c

```diff
@@ -4538,6 +4566,16 @@ static avifResult avifDecoderDecodeTiles(avifDecoder * decoder, uint32_t nextIma
     return AVIF_RESULT_OK;
 }
 
+static avifBool avifDecoderDataLastFrameFullyDecoded(const avifDecoderData * data)
```

Comment:
Nit: Last => Current ?

If "Current" is incorrect, then we may want to change "Last" to "Previous" or "Prev". "Last" can be ambiguous because it can mean "the one before the current one" or "final".

Reply:
Renamed to avifDecoderDataFrameFullyDecoded because indeed it's a bit complicated which frame we're talking about. Also added a common: "Returns AVIF_FALSE if there is currently a partially decoded frame."
Ideally I would prefer a comment of the type "Returns AVIF_TRUE if..." but I thought it was simpler to explain this way.